src/test/appdistro/requests.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { expect } from "chai";
+import * as sinon from "sinon";
+import { AppDistributionRequests } from "../../appdistribution/requests";
+import { FirebaseError } from "../../error";
+
+describe("distribution", () => {
+  const appId = "1:12345789:ios:abc123def456";
+  const distribution = new AppDistributionRequests(appId);
+
+  let sandbox: sinon.SinonSandbox;
+
+  beforeEach(() => {
+    sandbox = sinon.createSandbox();
+    sandbox.useFakeTimers();
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  describe("pollReleaseIdByHash", () => {
```

Comment:
_no_ other functions on this class deserve test cases? I think more test cases can/should be added. If you want to test out network requests, check out `nock`.